pkg/checkpoint/checkpointvaliditychecker/conservativecvc.go

```diff
@@ -0,0 +1,64 @@
+package checkpointvaliditychecker
+
+import (
+	es "github.com/go-errors/errors"
+
+	"github.com/filecoin-project/mir/pkg/checkpoint"
+	"github.com/filecoin-project/mir/pkg/crypto"
+	issconfig "github.com/filecoin-project/mir/pkg/iss/config"
+	checkpointpbtypes "github.com/filecoin-project/mir/pkg/pb/checkpointpb/types"
+	trantorpbtypes "github.com/filecoin-project/mir/pkg/pb/trantorpb/types"
+	tt "github.com/filecoin-project/mir/pkg/trantor/types"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+type ConservativeCVC struct {
+	issParams    *issconfig.ModuleParams
+	ownID        t.NodeID
+	hashImpl     crypto.HashImpl
+	chkpVerifier checkpoint.Verifier
+}
+
+// NewConservativeCVC returns a new ConservativeCVC. This checkpoint validity checker simply rejects checkpoints whose signatures cannot be verified because
```

Comment:
Try to keep the lines not longer than 120 characters.

Reply:
soz, done.